assets/queries/terraform/aws/secure_ciphers_disabled/query.rego

```diff
@@ -9,11 +9,15 @@ CxPolicy[result] {
 		"documentId": input.document[i].id,
 		"searchKey": sprintf("resource.aws_cloudfront_distribution[%s].viewer_certificate.minimum_protocol_version", [name]),
 		"issueType": "IncorrectValue",
-		"keyExpectedValue": sprintf("resource.aws_cloudfront_distribution[%s].viewer_certificate.minimum_protocol_version is TLSv1.1 or TLSv1.2", [name]),
-		"keyActualValue": sprintf("resource.aws_cloudfront_distribution[%s].viewer_certificate.minimum_protocol_version isn't TLSv1.1 or TLSv1.2", [name]),
+		"keyExpectedValue": sprintf("resource.aws_cloudfront_distribution[%s].viewer_certificate.minimum_protocol_version starts with TLSv1.1 or TLSv1.2", [name]),
+		"keyActualValue": sprintf("resource.aws_cloudfront_distribution[%s].viewer_certificate.minimum_protocol_version isn't start with TLSv1.1 or TLSv1.2", [name]),
 	}
 }
 
-checkMinProtocolVersion("TLSv1.1") = true
+checkMinProtocolVersion(startswith("TLSv1.1")) = true
```

Comment:
Hi @frediana,
This line is unnecessary and can be removed, this check is already being made in line 20.
The Unit Tests are failing because of incorrect syntax on this line  making this query fail its execution
```suggestion
```